src/client/package.json

```diff
@@ -51,7 +49,12 @@
     "postcss": "^8.4.38"
```

Comment:
Do we need these overrides anymore? This was put in due to these being dependencies of CRA and CRA not updating them to later versions without the security vulnerabilities. Maybe with next.js these are managed or not used anymore. Probably worth a check.